spec/views/item_seasons/show.html.erb_spec.rb

```diff
@@ -21,6 +21,6 @@
 
   it 'displays the produces' do
     render
-    expect(rendered).to match(/Carambola/)
+    expect(rendered).to match(/carambola/)
```

Comment:
**suggestion:** If we don't care about casing, we could use the `i` modifier:

https://rubular.com/r/OJoM3T6RfTN108

So it'd become `/carambola/i`

Reply:
Have you considered this one? :)

Reply:
whoops! forgot about this one 😅 added the`i` modifier